kubernetes/eks-setup.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
```

Comment:
It might be helpful to use `set -eo pipefail` with the Bash scripts. That way issues will be easier to debug as the execution will stop whenever unexpected errors occur.